### DISCUSSION TITLE
Looking for a new maintainer

### DISCUSSION THREAD

**Discussion by VPKSoft** [desjarlais/Scintilla.NET#78] [Announcements]
Hi,
I'm looking for someone to take over this project. I'm just currently too busy and my interests are elsewhere to keep maintaining this.
This also applies to the satellite assemblies:
* [ScintillaLexers](https://github.com/VPKSoft/ScintillaLexers) - A class library containing lexer definitions for the Scintilla.NET
* [VPKSoft.ScintillaUrlDetect](https://github.com/VPKSoft/VPKSoft.ScintillaUrlDetect) - A library to detect URLs with the ScintillaNET control
* [VPKSoft.ScintillaSpellCheck](https://github.com/VPKSoft/VPKSoft.ScintillaSpellCheck) - A spell checking library for the Scintilla.NET
* [ScintillaDiff](https://github.com/VPKSoft/ScintillaDiff) - A class library for comparing two text files with the Scintilla.NET control
* [ScintillaNetPrinting](https://github.com/VPKSoft/ScintillaNetPrinting) - Add print functionallity to Scintilla.NET

It has been fun for over three years with this project after continuing from the [original](https://github.com/jacobslusser/ScintillaNET), which was abandoned after Oct 31, 2018. That is why I wish to transfer the ownerships of these projects rather than leaving them unmaintained in silence.

Please reply to this message in public if you wish to continue with the project or some of these satellite assemblies 👍

**Comment by desjarlais** [comment]
I might be interested, let me take a look at the project and let you know.

**Reply by desjarlais** [reply]
I would like to take this project on, seems interesting enough for some of the stuff I want to work on, so I'll give it a try.  

How did you want to handle it from here?

**Reply by VPKSoft** [reply]
Hi,
I'll just transfer the ownership of the repository and create a new empty one redirecting to the right place. I'll do the transfer today or tomorrow.

**Reply by desjarlais** [reply]
That works for me.  Only Scintilla.NET or all of the other associated projects?

**Reply by jmairboeck** [reply]
Do forks stay in place (pointing to the new location) or must they be recreated after the transfer?

**Reply by desjarlais** [reply]
I think they will point to the new location:

"If the transferred repository has any forks, then those forks will remain associated with the repository after the transfer is complete."

https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository

**Comment by VPKSoft** [comment]
> That works for me. Only Scintilla.NET or all of the other associated projects?
I can transfer them all if you want them? Otherwise they are for other willing maintainers to take...

**Reply by desjarlais** [reply]
You can send them all.  Did you eventually want them all to be in Scintilla.NET or you wanted them to be separate projects for a reason?

Also, do you have any notes on the deployment process?  I've just been looking through native Scintilla source and your Scintilla.NET project, I haven't really looked at the build/deployment process for Scintilla.NET.

**Comment by VPKSoft** [comment]
Hi,
I activated the main repository transfer, all you need to do is accept.
The satellite assemblies should be in separate packages to avoid bloating the main package and code base.
The deployment is in the `./github/workflows/dotnet-nuget-release.yml` file.

**Reply by VPKSoft** [reply]
Seems that the redirect works automatically, I'll start transferring the satellite assemblies then 👍

**Reply by VPKSoft** [reply]
Okay, all the satellites should now be pending for transfer.

**Comment by VPKSoft** [comment]
All the repositories should now be transferred.

**Reply by jmairboeck** [reply]
@VPKSoft We are also using your version of [ScintillaNET-FindReplaceDialog](https://github.com/VPKSoft/ScintillaNET-FindReplaceDialog). Are you going to keep that?

**Reply by VPKSoft** [reply]
I missed that, there are so many, I activated the transfer now.
